Backend/UnitTesting/LoginManagerUT.cs

```diff
@@ -29,88 +35,126 @@ public void LoginCheckUserExists_Success_ReturnTrue()
         {
-
+            user.Disabled = true;
+            bool result = lm.LoginCheckUserDisabled(request);
+            Assert.AreEqual(true, result);
         }
 
         [TestMethod]
         public void LoginCheckUserDisabled_Fail_ReturnTrue()
         {
-
+            um.EnableUser(user);
+            bool result = lm.LoginCheckUserDisabled(request);
+            Assert.AreNotEqual(true, result);
         }
 
         [TestMethod]
         public void LoginCheckUserDisabled_Success_ReturnFalse()
         {
-
+            user.Disabled = false;
+            bool result = lm.LoginCheckUserDisabled(request);
+            Assert.AreEqual(true, result);
         }
 
         [TestMethod]
         public void LoginCheckUserDisabled_Fail_ReturnFalse()
         {
-
+            user.Disabled = true;
+            bool result = lm.LoginCheckUserDisabled(request);
+            Assert.AreNotEqual(false, result);
         }
 
         //LoginCheckPassword
 
         [TestMethod]
         public void LoginCheckPassword_Success_ReturnTrue()
         {
-
+            bool result = lm.LoginCheckPassword(request);
+            Assert.AreEqual(true, result);
         }
 
         [TestMethod]
         public void LoginCheckPassword_Fail_ReturnTrue()
         {
-
+            request.password = "pass";
+            bool result = lm.LoginCheckPassword(request);
+            Assert.AreNotEqual(true, result);
         }
 
         [TestMethod]
         public void LoginCheckPassword_Success_ReturnFalse()
         {
-
+            request.password = "pass";
+            bool result = lm.LoginCheckPassword(request);
+            Console.WriteLine(user.IncorrectPasswordCount);
+            Assert.AreEqual(false, result);
+            Assert.AreNotEqual(0, user.IncorrectPasswordCount);
         }
 
         [TestMethod]
-        public void LoginCheckPassword_Fail_ReturnFalse()
+        public void test()
         {
+            user = um.CreateUser("rrrcf2080@gmail.com", "qwerty12345", new DateTime(1996, 12, 15));
+            Console.WriteLine(user.IncorrectPasswordCount);
+            request.password = "pass";
+            lm.LoginCheckPassword(request);
+            lm.LoginCheckPassword(request);
+            lm.LoginCheckPassword(request);
+            Console.WriteLine(user.IncorrectPasswordCount);
+            Assert.AreEqual(0, user.IncorrectPasswordCount);
+        }
 
+        [TestMethod]
+        public void LoginCheckPassword_Fail_ReturnFalse()
+        {             
+            bool result = lm.LoginCheckPassword(request);
+            Assert.AreNotEqual(false, result);
         }
 
         // LoginAuthorized()
-
+        /*
         [TestMethod]
         public void LoginAuthorized_Success_ReturnToken()
         {
 
+            lm.LoginAuthorized();
         }
 
         [TestMethod]
-        public void LoginAuthorized_Fail_ReturnToken()
+        public void LoginAuthorized_Fail_ReturnNull()
```

Comment:
Again, modifying login code here. I believe this is from your merge. You'll have to wait until the login PR is merged before this PR is merged. Undoubtedly there will be changes you have to revert as well, since his PR has changed since you merged that branch into this one.